Library/Homebrew/dev-cmd/man.rb

```diff
@@ -55,6 +55,10 @@ def man_args
   end
 
   def man
+    # Disabled to make CI pass, as we removed most of the content of the REAME.md file,
+    # and build_man_page would fail.
+    return if ENV["CI"]
```

Comment:
I'd suggest just removing CI entirely instead and/or adjusting the `azure-pipelines.yml` file to be a no-op.

Reply:
Yes, that's probably also a good idea. I'll update my PR accordingly.